docs/src/content/docs/reference/frontmatter.md

```diff
@@ -146,3 +146,45 @@ title: Page with a custom last update date
 lastUpdated: 2022-08-09
 ---
 ```
+
+### `prev`
+
+**type:** `boolean | string | { href?: string; label?: string }`
+
+Overrides the [global `pagination` option](/reference/configuration/#pagination). If a string is specified, the generated link text will be replaced and if an object is specified, both the link and the text will can be overridden.
+
+```md
+---
+# Hide the previous page link
+prev: false
+---
+```
+
+```md
+---
+# Override the previous page link text
+prev: Continue the tutorial
+---
+```
+
+```md
+---
+# Override both the previous page link and text
+prev: 
+  href: /unrelated-page/
```

Comment:
I don’t know if this was the correct choice or not but in the `hero.actions` frontmatter and in the sidebar config we use `link` rather than `href`, so we should probably be consistent with this feature and use `link` here too.